app/assets/javascripts/modules/show-gov-uk-verify-hint.js

```diff
@@ -0,0 +1,66 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.ShowGovUkVerifyHint = function () {
+    this.start = function (element) {
+      if (window.location.href.indexOf("/personal-tax-account/sign-in/prove-identity") > -1) {
+        checkLastSuccessfulIdp(element)
+      }
+    }
+    
+    this.render = function (element, data) {
+      renderHint(element, data)
+    }
+
+    function checkLastSuccessfulIdp (element) {
+      $.ajax({
+        url: 'https://www.signin.service.gov.uk/successful-idp',
+        cache: false,
+        dataType: 'jsonp',
+        timeout: 3000
+      }).then(function(data){
+          renderHint(element, data);
+      }, function(e){console.log("error", e)})
+    }
+
+    function renderHint (element, data) {
+      if (data != null && data['found'] == 'true') {
+        $(element).html(generateHtml(data)).show()
+        $('button:contains("Continue")').addClass('govuk-button--secondary')
```

Comment:
The `:contains` pseudoselector seems brittle to me since the copy is not controlled by this script – if content will be updated this line will fail to execute.

Reply:
Yeah, there isn't an easier way and we don't expect the copy to change in the next 3 days anyway.